portal/src/main/webapp/js/src/study-view/component/MutatedGeneCNATable.js

```diff
@@ -346,7 +346,8 @@ var EnhancedFixedDataTableSpecial = (function() {
                             placeholder: this.props.hasOwnProperty('placeholder')?this.props.placeholder:"Search...",
                             "data-column": this.props.name,
                             value: this.state.key,
-                            onChange: this.handleChange})
+                            onChange: this.handleChange,
+                            title: this.props.hasOwnProperty('placeholder')?this.props.placeholder:"Search..."})
```

Comment:
@zhx828 I think I should revert this particular line, it will be fixed as part of pull request #1497.